docs/src/main/asciidoc/security-openid-connect-multitenancy.adoc

```diff
@@ -590,11 +604,11 @@ To configure the resolution of the tenant identifier, use one of the following o
 * <<default-tenant-resolver>>
 * <<annotations-tenant-resolver>>
 
-These tenant resolution options will be tried in turn, in the order they are listed, until the tenant id gets resolved.
-If the tenant id remains unresolved (`null`) in the end then the default (unnamed) tenant configuration will be selected.
+These tenant resolution options are tried in the order they are listed until the tenant ID gets resolved.
+If the tenant ID remains unresolved (`null`), the default (unnamed) tenant configuration is selected.
 
 [[tenant-resolver]]
-=== Resolve with `TenantResolver`
+=== Resolve with TenantResolver
```

Comment:
Sheila is highlighting such words representing classes or objects like UserInfo in her PRs, it looks OK in the highlighted form

Reply:
Updated. Thanks.